nextflow.config

```diff
@@ -232,8 +232,8 @@ manifest {
     description     = """Workflow converts one or multiple bam/cram files to fastq format"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=22.10.1'
-    version         = '2.0.0'
-    doi             = ''
+    version = '2.1.0dev'
```

Comment:
well this aligned is abyssmal :D But can be fixed later

Reply:
afaik this is due to the bump-version, this is always not aligned, I gave up there as well :D